src/payform.coffee

```diff
@@ -200,6 +200,8 @@
   reFormatCardNumber = (e) ->
     return if e.target.value is ""
     e.target.value = payform.formatCardNumber(e.target.value)
+    if e.target.value.indexOf('‎\u200e') == -1 and document.dir == 'rtl'
```

Comment:
I find that `document.dir == 'rtl'` is a much broader test we need to check for and also faster to short circuit, so given this is an `and` operation, I'd change this test to be the first one on the order